src/main/java/Duke.java

```diff
@@ -1,10 +1,148 @@
+import java.util.Scanner;
 public class Duke {
+    private static final int MAX_TASK_LENGTH = 100;
+    private static int taskCount = 0;
+
     public static void main(String[] args) {
+        //Print logo and opening message
+        printLogo();
+        printGreeting();
+
+        //Receive user input and execute corresponding functions
+        getUserInput();
+
+        //Print closing message
+        printFarewell();
+    }
+
+    public static void loopEcho() {
+        Scanner sc = new Scanner(System.in);
+        String userInput = sc.nextLine();
+        while (!userInput.equals("bye")) {
+            printDivider();
+            System.out.printf("\t%s%n", userInput);
+            printDivider();
+            userInput = sc.nextLine();
+        }
+    }
```

Comment:
I may be wrong but this method does not seem to be used anywhere in the code! If so, it can be deleted.